### DISCUSSION TITLE
Questions Re: Using Brunch on MS Surface Pro 5 Advanced (i.e., with LTE)

### DISCUSSION THREAD

**Discussion by yasurok** [sebanc/brunch#1988] [Q&A]
I would like to solicit help from you Brunch gurus.

I used to enjoy using Brunch on Microsoft SurfacePro 5 Advanced (2017). I could sudo in the terminal just fine, and my MPP 2.0 stylus/pen (Raphael 520C) worked with its pressure correctly picked up. I was very happy and very grateful to all the people who contributed to make this possible, especially sebanc.

Recently I tried to upgrade to r119, and that's when all the hell broke loose. I could not log on and even after a powerwash, it still did not let me log in saying cryptohome was broken. Oh, well -- time for a clean reinstallation.

I reinstalled r119 afresh with the Kench recovery. SP5 uses Core i5-7300U, which is used by HP Chromebox G2, which is codenamed Kench.

It ran fine and with the ipts framework option turned on, the touch screen works. However, my stylus does not work at all. I really need to be able to use this pen so this is a deal breaker. So here's my first question.

- Q1: What is the latest Brunch version where MPP styli still worked?

I was also bummed out to learn that I could no longer sudo after invoking Crosh and then "shell." Supposedly I am expected to invoke VT-2 for that but I am not sure how to do it on my SP5. Obviously I am supposed to "Ctrl + Alt + RightArrow" but it does not do anything. Hence my second question.

- Q2: Is it possible to sudo like the way I used to be able to? If I have to invoke VT-2, how do I do that on my SP5. I do not want to install a full-fledged Linux distro; I just want to use Chromebrew and install a few utilities.

I have downgraded to r114, which gave me sudo again, but not stylus support.

I'd appreciate your input very much! Thanks!

**Comment by yasurok** [comment]
I reverted back to r107 and it works perfectly on both points. That's great, but I would like to enjoy newer versions too.  :)

**Reply by maxed4901** [reply]
Kinda late to the party but did u update or upgrade later? 

Google pixelbook 2017 has eve. Also 7th gen i5.

 And samsung chomebook pro xe510 has coraline which is m3 6th gen. 

Should work for your device. 

From what i seen, brunch is not the main issue. Chrome os file creates the issues.
